### DISCUSSION TITLE
different output locally

### DISCUSSION THREAD

**Discussion by davidsonsns** [orval-labs/orval#131] [Q&A] [answered]
branch: `next`
version: `5.3.0`

After cloning the repository I tried to make a test generating a react-query code with exactly the same `react-app-with-react-query` sample.

the output was different. as we can see below.
but, my main point is "how can I generate the code with a condition like `{ enabled: !!(version && petId), ...queryConfig }`"?

should I add an extra field to `output.override.query` in the `orval.config.js` file?

ie:
![Screen Shot 2021-05-18 at 22 45 21](https://user-images.githubusercontent.com/9276511/118744909-8d614a00-b82b-11eb-9ad5-9a67d4c97fc8.png)

**Comment by anymaniax** [comment, marked as answer]
Hello, it was a bug fix with the `5.3.1` and the next tag is the rc. I still need to update the doc and examples.

**Reply by davidsonsns** [reply]
lol so fast man! thank you so so much. WOO! and amazing lib.
looking forward to helping you contributing with anything ASAP

**Reply by anymaniax** [reply]
with pleasure! Don't hesitate if you need help or If you have a problem 🚀

**Reply by spaceo** [reply]
@anymaniax @davidsonsns 
I cannot see anything about this subject in the documentation.
How should the configuration look like in order to add "enabled" to the options?
